hr_timesheet_auto_creation/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# © 2016-2017 Elico Corp (https://www.elico-corp.com)
+# Copyright 2019 Coop IT Easy SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "HR TimeSheet Auto Creation",
+    "summary": "Create weekly timesheets for employees automatically",
+    "version": "12.0.1.0.0",
+    "category": "Human Resources",
+    "author": (
+        "Coop IT Easy SCRLfs, Elico Corp, Odoo Community Association (OCA)"
```

Comment:
should the oca really be there? it has been added when we backported the module from 10.0.

Reply:
Did we backport it from OCA or Elico Corp?

Reply:
i didn’t find this module in the [timesheet oca repo](https://github.com/OCA/timesheet), so i suppose it came from elico corp directly. @vvrossem?

Reply:
Backported from [Elico-Corp](https://github.com/Elico-Corp/odoo-addons/tree/10.0/hr_timesheet_auto_creation) with the purpose to submit it to OCA (didn't happen)